tests/visualization_tests/test_timeline.py

```diff
@@ -0,0 +1,61 @@
+import datetime
+from io import BytesIO
+from typing import List
```

Comment:
Could you replace `typing.List` with `list` as can be seen in https://github.com/optuna/optuna/commit/76c745be11edfa944c435b67789a2ce545615791?

We need `from __future__ import annotations` at the first line of this file.